requirements/edx/doc.txt

```diff
@@ -983,7 +983,7 @@ openedx-filters==1.10.0
     #   -r requirements/edx/base.txt
     #   lti-consumer-xblock
     #   ora2
-openedx-learning==0.13.1
+openedx-learning @ git+https://github.com/open-craft/openedx-learning@navin/component-collection-api
```

Comment:
`TODO`: update this after merging https://github.com/openedx/openedx-learning/pull/238 releasing new version of openedx-learning